src/coreclr/tests/src/profiler/eventpipe/eventpipe.cs

```diff
@@ -62,21 +64,26 @@ public static int RunTest()
 
                         if (traceEvent.EventName == "AllTypesEvent")
                         {
-                            success = ValidateAllTypesEvent(traceEvent);
+                            success |= ValidateAllTypesEvent(traceEvent);
```

Comment:
```suggestion
                            success &= ValidateAllTypesEvent(traceEvent);
```

I assume you success should become false if Validate returns false? Same issue a few more times below